tests/providers/azure/services/sqlserver/sqlserver_auditing_retention_90_days/sqlserver_auditing_retention_90_days_test.py

```diff
@@ -0,0 +1,150 @@
+from unittest import mock
+from uuid import uuid4
+
+from azure.mgmt.sql.models import ServerBlobAuditingPolicy
+
+from prowler.providers.azure.services.sqlserver.sqlserver_service import SQL_Server
+
+AZURE_SUSCRIPTION = str(uuid4())
```

Comment:
Remember that this value is present in the fixtures.

Reply:
Yes! Nice catch

Reply:
Done